### PR TITLE
Describe the change of forceId

### PR DESCRIPTION
**Pull request by jannyHou**
Connect to: strongloop/loopback#2642

Add new entry in 3.x-release-notes to notify users the change of forceId.
According to issue https://github.com/strongloop/loopback/issues/2642#issuecomment-247348380
Related pr: https://github.com/strongloop/loopback-datasource-juggler/pull/982

### CODE REVIEW

**Comment by jannyHou** [comment]
@bajtos PTAL. Thanks.

File: 3.0-RELEASE-NOTES.md [hunk 1]
```diff
@@ -154,3 +154,18 @@ var johndoe = new User({ name: 'John doe', age: 15, gender: 'm'});
     ```
 
 See [related code change](https://github.com/strongloop/loopback-datasource-juggler/pull/1084) here.
+
+## Defautly users could not set id if it is auto-generated
```
Comment (bajtos) [inline 3.0-RELEASE-NOTES.md]: `Users cannot provide a custom id value when id is auto-generated`

File: 3.0-RELEASE-NOTES.md [hunk 2]
```diff
@@ -154,3 +154,18 @@ var johndoe = new User({ name: 'John doe', age: 15, gender: 'm'});
...
+
+The default value of config variable `forceId` is set to `true` if a model uses auto-generated id. It means users cannot assign the id value to a model instance but use the auto-generated one instead.
```
Comment (bajtos) [inline 3.0-RELEASE-NOTES.md]: Please wrap the long line at 80 chars.

**Comment by bajtos** [comment]
@jannyHou you can also use my text from the migration guide for inspiration, see https://github.com/strongloop/loopback.io/blob/de428548c2760b2ba5a3015ca6a9e478f564ff91/pages/en/lb3/Migrating-to-3.0.md#models-with-auto-generated-ids-reject-user-provided-id-values

**Comment by jannyHou** [comment]
Thank you lol. 🚢

**Pull request merged**